shopping-cart/shopping-cart.js

```diff
@@ -0,0 +1,15 @@
+import cart from '../data/cart.js';
+import books from '../data/cart.js';
+import renderLineItems from './render-line-items.js';
+import { findById, calcLineItem } from '../common/utils.js';
+
+export function shoppingCart() {
```

Comment:
It's ok to write this as a function, but then you need to call the function in this file in order to get it to actually render stuff to the page.